package.json

```diff
@@ -0,0 +1,28 @@
+{
+  "name": "loopback-sdk-ios-codegen",
+  "version": "0.0.0",
+  "description": "Tool for auto-generating iOS client source code for LoopBack App",
+  "keywords": [ "StrongLoop Labs", "LoopBack", "client", "SDK", "iOS", "codegen" ],
+  "bin": {
+    "codegen-ios": "bin/codegen-ios"
```

Comment:
The Angular SDK uses `lb-ng` command, please use the `lb-` prefix here too. I am personally fine with the short name `lb-ios`, although `lb-codegen-ios` is ok too.